inference-engine/src/mkldnn_plugin/nodes/mkldnn_log_softmax_node.cpp

```diff
@@ -41,21 +37,45 @@ MKLDNNLogSoftmaxNode::MKLDNNLogSoftmaxNode(const std::shared_ptr<ngraph::Node>&
         IE_THROW() << "Operation with name '" << op->get_friendly_name() <<
             "' is not an instance of LogSoftmax from opset5.";
 
-    if (getOriginalInputsNumber() != 1 || getOriginalOutputsNumber() != 1)
+    if (inputShapes.size() != 1 || outputShapes.size() != 1)
         IE_THROW() << errorPrefix << " has incorrect number of input/output edges!";
 
-    SizeVector dims = op->get_input_shape(0);
-    if (!dims.size())
+    VectorDims dims = getInputShapeAtPort(0).getDims();
```

Comment:
looks like enough to take input shape rank because below we use `dims.size()` only